_maps/map_files/RandomRuins/LavaRuins/lavaland_surface_althland_facility.dmm

```diff
@@ -0,0 +1,2589 @@
+//MAP CONVERTED BY dmm2tgm.py THIS HEADER COMMENT PREVENTS RECONVERSION, DO NOT REMOVE
+"aa" = (
+/obj/effect/landmark/burnturf,
+/turf/simulated/floor/plating/airless,
```

Comment:
This is going to cause active turfs during init, you're going to need to change over these turfs to respect lavaland air

Reply:
yepppp, give me a bit

Reply:
done